### DISCUSSION TITLE
for [ordered_json], the member function [parse] do not support keeping order of the input

### DISCUSSION THREAD

**Discussion by sinbobo** [nlohmann/json#2539] [Q&A] [answered]
`std::string input;// input json`
`{// I want to keep order,but failed`
`    ordered_json json = json::parse(input);`
`    ....`
`}`
`{// another way to cover this problem, but ugly...`
`    std::stringstream str(input);`
`    ordered_json jsonInput;`
`    str >> jsonInput;`
`    ....`
`}`
hope someday, nlohmann/json can support this

**Comment by nlohmann** [comment, marked as answer]
Can you try calling ordered_json::parse?

**Comment by sinbobo** [comment]
sorry, just forgive my stupid....&nbsp;


 
---Original---
